### PR TITLE
En 1567/implement calculation moodal

### PR DESCRIPTION
**Pull request by stkane**
Implements calculation modal [en-1567](https://arcadiapower.atlassian.net/jira/software/projects/EN/boards/50?selectedIssue=EN-1567). Realllly basic looking, but hopefully clear. I might revisit some of the styling if there's additional time.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+button {
+  height: 25px;
+}
```
Comment (stkane) [inline src/index.css:3]: planning to add more global styles for error handling.

File: src/components/utility-statement-element.jsx [hunk 2]
```diff
@@ -37,15 +38,20 @@ const UtilityStatementElement = ({ arcUtilityStatement }) => {
...
+          <button onClick={closeModal}>close</button>
+        </div>
+        <>
+          {
+            counterFactualResults ? <div style={containerStyle}>
+              <CounterfactualResults title="Current Cost" results={counterFactualResults.currentCost}></CounterfactualResults>
+              <CounterfactualResults title="Current Cost Without Solar" results={counterFactualResults.currentCostWithoutSolar}></CounterfactualResults>
+            </div>
+              : <p>Loading...</p>
+          }
```
Comment (stkane) [inline src/components/utility-statement-element.jsx:53]: Will revise this slightly for error handling.

**Review by xelzi — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
     "cors": "^2.8.5",
     "dayjs": "^1.11.5",
     "humps": "^2.0.1",
+    "normalize.css": "^8.0.1",
```
Comment (xelzi) [inline package.json:16]: is this being used anywhere?

Comment (stkane) [reply package.json:16]: Ah whoops, will remove!

**Pull request merged**